### DISCUSSION TITLE
as udp proxy

### DISCUSSION THREAD

**Discussion by DrOreo002** [xjasonlyu/tun2socks#85] [Q&A]
Hey, I'm currently developing bots with https://github.com/lsalzman/enet as its network library, wondering if its possible to use tun2socks with that library. Sorry if this looks stupid, I don't have that much knowledge with sockets stuff

**Comment by xjasonlyu** [comment]
Sorry, but I don't quite understand. What is that library for and why do you need tun2socks as its library or as your bot's library?

**Reply by DrOreo002** [reply]
Hmm I'll find a better way to explain it xd, sorry if I bothers you and thank you for still replying though

**Reply by xjasonlyu** [reply]
No problem
